bom/runtime/pom.xml

```diff
@@ -733,6 +733,11 @@
                 <artifactId>quarkus-kubernetes</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-openshift</artifactId>
```

Comment:
would this be better named as quarkus-openshift-s2i as we wouldn't have this if s2i wasn't in the picture and we could imagine more and more users using openshift without needing s2i at all....so maybe down the line there is something a bit more apropriate to be named quarkus-openshift ?

Reply:
I am not a big fan of `quarkus-openshift-s2i`. Mostly, because it's not just about s2i but also to provide default properties for openshift e.g.`quarkus.kubernetes.deploymenet-target` etc. In the future we could move all the openshfit  related code, in this extension too (if there wasn't time constraints I would have done it already).

Reply:
ack -makes sense. I guess question is if s2i.enbled=true  or s2i.enabled=false is the default then ?

Reply:
`s2i.enabled=true` by default.